docs/quickstarts/javascript/react/application-setup.mdx

```diff
@@ -0,0 +1,154 @@
+---
+sidebar_label: Application setup
+---
+
+# React Application setup
+
+We’re going to build a very bare bones application for this tutorial. We’ll start by creating an application using the create-react-app generator: In your terminal, execute the following command:
+
+```
+npx create-react-app aserto-react-demo
+```
+
+You can now cd aserto-react-demo and start the app by running:
+
+```
+yarn start
+```
+
+The familiar React logo should appear, indicating that the app is ready to go.
+
+## Adding OIDC dependencies
+
+Now that we have a running React application, we'll continue by installing and then importing the required dependency - `oidc-react`
+
+In your terminal, execute the following command:
+
+```
+yarn install oidc-react
+```
+
+The following environment variables are used to point your application to Aserto’s demo IDP, so that you don’t have to set one yourself. Create a file called `.env` and add the following:
+
+```
+REACT_APP_OIDC_DOMAIN=acmecorp.demo.aserto.com
+REACT_APP_OIDC_CLIENT_ID=acmecorp-app
+REACT_APP_OIDC_AUDIENCE=acmecorp-app
+REACT_APP_API_ORIGIN=http://localhost:8080
+```
+
+:::caution note
+Make sure the `.env` file is added to the `.gitignore` file so that it is not checked in.
+:::
+
+Open the file `index.js` and add the dependency:
+
+```javascript
+import { AuthProvider } from "oidc-react";
+```
+
+Add the following configuration object:
+
+```javascript
+const configuration = {
+  authority: `https://${process.env.REACT_APP_OIDC_DOMAIN}/dex`,
+  clientId: process.env.REACT_APP_OIDC_CLIENT_ID,
+  autoSignIn: true,
+  responseType: "id_token",
+  scope: "openid profile email",
+  redirectUri: window.location.origin,
+  audience: process.env.REACT_APP_OIDC_AUDIENCE,
+  onSignIn: () => {
+    window.location.replace(window.location.origin);
+  },
+};
+```
+
+Next, we'll wrap the top level React Application component with the AuthProvider, and pass it the required configuration we created.
+
+```javascript
+ReactDOM.render(
+  <React.StrictMode>
+    <AuthProvider {...configuration}>
+      <App />
+    </AuthProvider>
+  </React.StrictMode>,
+  document.getElementById("root")
+);
+```
+
```

Comment:
Note to Roie: once you do this, if you have "yarn start" running, the app will immediately put up the Dex authentication screen. This may be a good opportunity to mention this, let the reader know the email and password, and show that with those user credentials, you can now authenticate into the app using the Acmecorp demo identity provider. 

It would be cool for the developer to see the spinning atom screen as an intermediate step, before you have them replace the app with something more useful, as per below.